EyeWitness.py

```diff
@@ -202,6 +203,10 @@ def create_cli_parser():
         parser.print_help()
         sys.exit()
 
+    if not all((args.vhost_name, args.single, args.headless)):
```

Comment:
There's a bug here with the if statement.  This is causing the program to exit if only providing a file and telling it to use headless (which is a legitimate use) without specifying vhost_name.

Ex: ./EyeWitness.py -f urls.txt --headless
This errors out.  You'll need to just change the if statement to first detect if a vhost is being specified, and if so, make sure it has its requirements (headless and single).